### PR TITLE
Add section for triggering pipelines with parameters using the UI

### PR DESCRIPTION
**Pull request by olukotun-ts**
# Description
This pull request adds documentation for triggering pipelines with parameters using the UI.

# Reasons
I added this section because customers have asked about this feature especially when migrating from Jenkins where it is supported. CircleCI also supports this feature but it's not covered in our docs.

### CODE REVIEW

**Review by thomasr8 — APPROVED** [review]

**Comment by devinmarieb** [comment]
@olukotun-ts This wasn't building so I made a small change to force it to build. There is an issue with Datadog which is why it failed. Since this is a forked branch, I don't have permissions to try and re-run the build. If you could try that would be great. We can't merge this until it passes.

**Comment by olukotun-ts** [comment]
Thanks @devinmarieb I've kicked off the re-run.

**Comment by rosieyohannan** [comment]
@olukotun-ts just to note we do have this elsewhere in docs: https://circleci.com/docs/2.0/pipelines/#run-a-pipeline-from-the-circleci-app but the steps you provide are clearer so thank you!

File: jekyll/_cci2/pipeline-variables.md [hunk 1]
```diff
@@ -105,6 +105,21 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
 }' https://circleci.com/api/v2/project/:project_slug/pipeline
 ```
 
+### Passing parameters when triggering pipelines using the UI
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md:108]:
```suggestion
### Passing parameters when triggering pipelines using the CircleCI web app
```

File: jekyll/_cci2/pipeline-variables.md [hunk 2]
```diff
@@ -105,6 +105,21 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
...
+{: #passing-parameters-when-triggering-pipelines-using-the-ui }
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md:109]:
```suggestion
{: #passing-parameters-when-triggering-pipelines-using-the-circleci-web-app }
```

File: jekyll/_cci2/pipeline-variables.md [hunk 3]
```diff
@@ -105,6 +105,21 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
...
+
+In addition to using the CLI and API, you can also trigger a pipeline with parameters from the UI. To do this:
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md:111]:
```suggestion
In addition to using the CLI and API, you can also trigger a pipeline with parameters from the CircleCI web app. To do this:
```

File: jekyll/_cci2/pipeline-variables.md [hunk 4]
```diff
@@ -105,6 +105,21 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
...
+
+  1. Navigate to the dashboard view in the UI.
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md:113]:
```suggestion
  1. Navigate to the dashboard view in the web app.
```

File: jekyll/_cci2/pipeline-variables.md [hunk 5]
```diff
@@ -105,6 +105,21 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
...
+  2. Use the project filter to select the desired project.
+  3. Use the branch filter to select the branch on which you want to run the new pipeline.
+  4. Click the **Trigger Pipeline** button (towards the top right corner of the page).
+  5. Use the **Add Parameters** dropdown to specify the type, name, and value of your desired parameters.
+  6. Click **Trigger Pipeline**.
+
+**NOTE:** Please note that if you pass a parameter when triggering a pipeline via the UI, and the parameter has not been declared in the configuration file, the pipeline will fail with the error `Unexpected argument(s)`)
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md:120]:
```suggestion
**NOTE:** If you pass a parameter when triggering a pipeline from the web app, and the parameter has not been declared in the configuration file, the pipeline will fail with the error `Unexpected argument(s)`)
```

**Comment by olukotun-ts** [comment]
Thanks @rosieyohannan!

**Comment by rosieyohannan** [comment]
@olukotun-ts I pushed some changes to try and get this building but I think actually you will need to pull changes from master into your branch. Then hopefully it will build!

**Comment by olukotun-ts** [comment]
Thanks @rosieyohannan! I've pulled the updates and kicked off a new build.

**Comment by rosieyohannan** [comment]
Thank you @olukotun-ts!

**Pull request merged**